conf/cmi/core.entity_view_display.paragraph.target_group_link_item.default.yml

```diff
@@ -1,4 +1,3 @@
-uuid: 5c07c500-20d9-4232-a1f1-b83c57fa1a53
```

Comment:
Why are these uuid:s being removed? Should they be?

Reply:
These are related to some other update from platform config iirc.